base/applications/taskmgr/procpage.c

```diff
@@ -1248,8 +1248,11 @@ void ProcessPage_OnOpenFileLocation(void)
 
     StringCchPrintfW(pszCmdLine, dwLength, szCmdFormat, pszExePath);
 
-    /* Call the shell to open the file location and select it */
-    ShellExecuteW(NULL, L"open", L"explorer.exe", pszCmdLine, NULL, SW_SHOWNORMAL);
+    /* Call the shell to open the file location and select it. If explorer shell
+     * is not available, use ReactOS's alternative file browser instead. */
+    ShellExecuteW(NULL, L"open",
+                  GetShellWindow() ? L"explorer.exe" : L"filebrowser.exe",
+                  pszCmdLine, NULL, SW_SHOWNORMAL);
```

Comment:
Does this require using filebrowser.exe from ReactOS in Windows 2003?

Reply:
Yes, because Win2k3 explorer and also ReactOS don't support `/separate` option from the command line.

Reply:
Win2k3 taskmgr doesn't have this functionality (this feature is inspired by Win7+ but we implemented it in the Win2k3/XP-compatible way).

Reply:
I'd feel a lot better about approving this if there was a comment here saying something like:
```
/* FIXME: Use "explorer.exe /separate" instead of "filebrowser.exe" when implemented. */
```